models/fetchReviews-model.js

```diff
@@ -0,0 +1,79 @@
+const db = require("../db/connection.js");
+
+exports.fetchReviews = () => {
+  return db
+    .query(
+      `
+      SELECT 
+      reviews.title,
+  reviews.designer,
+  reviews.owner,
+  reviews.review_img_url,
+  reviews.category,
+  reviews.created_at,
+  reviews.votes,
+  reviews.review_id,
+  COUNT(comments.comment_id) AS comment_count
+      FROM reviews
+      LEFT JOIN comments
+      ON reviews.review_id = comments.review_id
+      GROUP BY reviews.review_id
+      ORDER BY created_at DESC;
+  `
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
```

Comment:
Nice work on the query! By default this will return the count as a `string`, if you want it to return in the form of a number then checkout `CAST` on sql docs